doc/api/dgram.md

```diff
@@ -95,6 +95,21 @@ Tells the kernel to join a multicast group at the given `multicastAddress` and
 one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
+### socket.addSourceSpecificMembership(sourceAddress, groupAddress[, multicastInterface])
+<!-- YAML
+added: vX.Y.Z
```

Comment:
nit: `added: REPLACEME`

Reply:
fixed in 305693a7b5bea1fa86eee5ca12376cc9753b21b6